traffic_ops/testing/api/README.md

```diff
@@ -41,7 +41,31 @@ In order to run the tests you will need the following:
     and here: `traffic-ops-test.conf` 
 
     The Traffic Ops users will be created by the tool for accessing the API once the database is accessible.
-
+    
+    Note that for the database to successfully set up the tables and run the migrations, you will need `goose`.
+    On your local machines(Macs), DO NOT install `goose` using `homebrew`. It will install a different version
+    `goose`. Instead, install it using the script located at
+    `traffic_ops/install/bin/install_goose.sh`. 
+    To test if `goose` migrated everything correctly, you can run the following command from the `traffic_ops/app`
+    directory:
+    `goose -env=test status "user=traffic_ops dbname=to_test sslmode=disable"`
+    
+    The result should be something similar to:
+    `goose: status for environment 'test'
```

Comment:
I think this output should be wrapped in triple backticks like this:

\```
asdf
\```

otherwise it doesn't render as preformatted text

Reply:
Fixed.